downstream/assemblies/hub/assembly-validated-content.adoc

```diff
@@ -22,55 +22,42 @@ Possible values are `certified` or `validated`.
-| Ansible | `network.acls` | A validated content collection to configure acls and provide capabilities to do operational state/healthchecks | Red Hat Ansible
-| Ansible | `network.interfaces` | A validated content collection to configure interfaces and provide capabilities to do operational state/healthchecks | Red Hat Ansible
-| Ansible | `network.ospf` | A validated content collection to configure ospf and provide capabilities to do operational state/healthchecks | Red Hat Ansible
-| Ansible | <name yet to decide> | Connectivity between on-prem network device (for ex CSR) and cloud gateway(for ex AWS) | Red Hat Ansible
-| Ansible | <name yet to decide> | A validated content (potentially a role) on Inventory Report that returns statistics and IDs of different edge nodes and servers |Red Hat Ansible
-| Ansible | | Network device Inventory report using html | Red Hat Ansible
-| Ansible | | Network config backup | Red Hat Ansible
-| Ansible | | Network config restore | Red Hat Ansible
-| Ansible | | IOS Updater | Red Hat Ansible
-| Ansible | | NXOS Updater | Red Hat Ansible
-| Ansible | | EOS Updater | Red Hat Ansible
-| Ansible | | Firewall policy automation - A validated content to take care of FW policy hygiene | Red Hat Ansible
-| Ansible | | OSbuilder for RHEL Edge disconnected (customer request) | Red Hat Ansible
-| Ansible | | Middleware collection | Red Hat Ansible
-| Ansible | | Windows and Linux Compliance |Red Hat Ansible
-| Ansible | | SAP Deployment | Red Hat Ansible
-| Ansible | | Automation controller configuration | Red Hat Ansible
-| Ansible | | Execution Environment Utilities | Red Hat Ansible
-| Ansible | | {HubNameStart} configuration | Red Hat Ansible
-| Ansible | | AAP utilities | Red Hat Ansible
-| Ansible | | Role to deploy and migrate a web application on _{AWS}_ (AWS) | Red Hat Ansible 
-| Ansible | | Role to deal with AWS orphaned instances by tag | Red Hat Ansible
-| Ansible | | Role to create a customized _Amazon Machine Images_ (AMI) | Red Hat Ansible
-| Ansible | | Role to detach and delete AWS _Internet Gateway_ (IGW)s |Red Hat Ansible
-| Ansible | | Role to configure a multi-region CloudTrail | Red Hat Ansible
-| Ansible | | Role to configure CloudTrail encryption | Red Hat Ansible
-| Ansible | | Role to troubleshoot EC2 instances failing to join an ECS cluster | Red Hat Ansible
-| Ansible | | Role to troubleshoot _Relational database Service_ (RDS) connectivity from an instance | Red Hat Ansible
-| Ansible | | Role to troubleshoot _Virtual Private Cloud_ (VPC) connectivity issues | Red Hat Ansible
-|====
+Upload the content and define the variables using the command:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+ansible-playbook collection_seed.yml 
+-e automationhub_admin_password=<admin password> 
+-e automationhub_main_url=<main_url>
+-e automationhub_api_token=<api_token>
+ansible-playbook collection_seed.yml -e @<name of file where variables are defined>
```

Comment:
Since either password or token should be used, this example should only use either one, not both. Please add `automation_hub_require_content_approval` as well.

Also, in this example, all variables are defined with `-e <var>=<value>` so no need to mention about separate variable file and `-e @<name of file where variables are defined>` can be removed.